geode-core/src/main/java/org/apache/geode/management/internal/operation/RestoreRedundancyPerformer.java

```diff
@@ -112,14 +114,14 @@ public RestoreRedundancyResults perform(Cache cache, RestoreRedundancyRequest op
     return finalResult;
   }
 
-  // this returns either an Exception or RestoreRedundancyResults
+  // this returns RestoreRedundancyResults or null based on
```

Comment:
Incomplete comment here?

Reply:
Good catch. Task switching is bad. The comment just restates what is visible in the code so I deleted it.

Reply:
Thanks for your attention to detail.